bouncy/distribution.yaml

```diff
@@ -383,6 +383,17 @@ repositories:
       url: https://github.com/eProsima/Fast-RTPS.git
       version: master
     status: developed
+  gazebo_ros_pkgs:
+    doc:
+      type: git
+      url: https://github.com/ros-simulation/gazebo_ros_pkgs.git
+      version: bouncy
```

Comment:
Travis is failing because this branch doesn't exist. I assume you want the `ros2` branch here instead of Bouncy.
(also we don't have doc jobs on the ROS2 farm yet so this section will have no effect but it doesn't hurt to keep it)